src/Gui/SoFCCSysDragger.cpp

```diff
@@ -88,6 +88,8 @@
 
 using namespace Gui;
 
+constexpr char* TDraggerCylinderShape{"CSysDynamics_TDragger_CylinderShape"};
```

Comment:
constexpr just specifies that the pointer won't change, just like if you would have written `char * const TDraggerCylinderShape`.

I would suggest that you write:
```suggestion
constexpr const char* TDraggerCylinderShape = "CSysDynamics_TDragger_CylinderShape";
```